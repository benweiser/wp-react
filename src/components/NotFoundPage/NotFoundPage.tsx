import * as React from "react";

export interface NotFoundPageProps {
  readonly location: string;
}

const NotFoundPage: React.SFC<NotFoundPageProps> = (): JSX.Element => {
  return (
    <div>
      <h1>This page could not be found. No match for {location.pathname}</h1>
    </div>
  );
};

export default NotFoundPage;
