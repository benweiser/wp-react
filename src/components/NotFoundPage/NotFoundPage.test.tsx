import { shallow } from "enzyme";
import NotFoundPage from "./NotFoundPage";
import * as React from "react";
import toJson from "enzyme-to-json";

/*it("renders 404 message", () => {
  const wrapper = shallow(<NotFoundPage location={"/404"} />);
  const content = (
    <h1>This page could not be found. No match for {location.pathname}</h1>
  );
  expect(wrapper.contains(content)).toEqual(true);
});*/

it("404 component renders correctly", () => {
  const wrapper = shallow(<NotFoundPage location={"/404"} />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
