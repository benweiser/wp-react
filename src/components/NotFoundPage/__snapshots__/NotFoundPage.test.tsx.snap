// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 component renders correctly 1`] = `
<div>
  <h1>
    This page could not be found. No match for 
    /
  </h1>
</div>
`;
